### ISSUE TITLE
Consider bundling this workflow with alfred-airpodspro-connector together and add noise cancel control

### ISSUE THREAD

**Issue by reorx** [godbout/alfred-airpodspro-battery#6]
Then the new workflow will have three commands:
- `apc` for connecting the AirPod
- `apb` for reading the battery info
- `apn <mode>` for switching to a noise cancel mode.

**Comment by godbout** [comment]
yeah, [Vitor kinda had the same idea](https://www.alfredforum.com/topic/14794-alfred-airpods-pro-connector/?do=findComment&comment=111242), combining both the connect/disconnect and battery Workflows.

thing is 1) that'll make me need to dig again into something i haven't touched for a long long time and that for me works well lol 2) i think at least the battery Workflow is full of scary bash scripting that i tend to forget and have a hard time getting back into.

so the idea is nice, but i'm pretty sure i'm not gonna dig into it soon. keeping it open tho. thanks!

**Comment by reorx** [comment]
Totally understand, I also have open-source projects that I cannot spend much time on them, I'm here to jot it down for your reference when some day in the future you may feel like working on it
